addons/osio-addon/src/main/java/io/fabric8/launcher/osio/steps/OpenShiftSteps.java

```diff
@@ -103,6 +107,10 @@ public ConfigMap createJenkinsConfigMap(OsioProjectile projectile, GitRepository
         return cm;
     }
 
+    public void ensureJenkinsCDCredentialCreated() {
+        jenkinsGitCredentials.ensureCredentials(gitService.getLoggedUser().getLogin());
+    }
```

Comment:
Jenkins in OpenshiftSteps? Looks like this shouldn't be done here. How about a JenkinsSteps?